firmware/misaligned_op/main.rs

```diff
@@ -8,42 +7,98 @@ use miralis_abi::{log, setup_binary, success};
 
 setup_binary!(main);
 
+// Define a macro to handle misaligned load/store operations
+macro_rules! misaligned_op {
+    ($asm_op:expr, $error_code:expr, $addr:expr, $value:expr) => {
+        unsafe {
+            asm!(
+                $asm_op,
+                addr = in(reg) $addr,
+                r = inout(reg) $value => _,
+            );
+        }
+    };
+}
```

Comment:
What do you gain with this macro? You save `addr = in(reg)` and `r = inout(reg)` only as you need to write the assembly snippet right?

What is the `error_code` about? It doesn't seem used

Overall this macro seems more confusing than it helps.